### PR TITLE
bug/Issue-91: Refactor headers styles to prevent logo from shrinking

### LINKED ISSUE #91: greenwood logo is inconsistently sized on medium / table breakpoints
## Summary

Compared to other break points, especially at an iPad Prop breakpoint, the logo in the header can get really small or really close to the navigation, instead of being "casually" off to the left

Screenshot 2024-09-05 at 6 46 46 PM
Screenshot 2024-09-05 at 6 46 55 PM

## Details

For example, on desktop and phone breakpoints, it all looks a little more proportional
Screenshot 2024-09-05 at 6 46 37 PM
Screenshot 2024-09-05 at 6 47 04 PM
Screenshot 2024-09-05 at 6 46 14 PM

### PR DESCRIPTION
**Pull request by DevLab2425**
## Related Issue
Fix #91 

## Summary of Changes

- Update header styles to allow the logo to remain at a reasonable, legible size. 

https://github.com/user-attachments/assets/b130b72c-4151-4569-abac-7aab956c81f8

### CODE REVIEW

File: src/components/header/header.module.css [hunk 1]
```diff
@@ -137,8 +136,9 @@
 }
 
 @media screen and (min-width: 768px) {
-  .logoLink svg.greenwood-logo-full {
-    width: 60%;
+  .navBar {
```
Comment (thescientist13) [inline src/components/header/header.module.css:139]: I might consider sizing it down a "little" bit between the 768px - 1024px range, as it does seem to dominate a bit of the header in that gap.
Screenshot 2024-10-15 at 4 12 18 PM


Maybe we can `clamp` it or put a `max-width` or something on it?

**Review by thescientist13 — COMMENTED** [review]
Nice, I think we can make one small little tweak but otherwise this looks good!

File: src/components/header/header.module.css [hunk 2]
```diff
@@ -137,17 +145,21 @@
...
+    display: flex;
```
Comment (thescientist13) [inline src/components/header/header.module.css:149]: I _think_ we might want keep it as `display: contents` at this break point as well, as the vertical alignment of the navigation looks slightly better aligned vertically with the logo and mobile menu icon (IMO 😅 )

#### `display: flex` (current)
Screenshot 2024-10-16 at 9 38 26 PM

#### `display: contents`
Screenshot 2024-10-16 at 9 39 30 PM

Comment (DevLab2425) [reply src/components/header/header.module.css]: Great catch.

Comment (thescientist13) [reply src/components/header/header.module.css]: Hmm, now the nav links stay in the center as we get past tablet breakpoints, as opposed to staying to the right, against the side nav
Screenshot 2024-10-17 at 11 30 14 AM

Apologies for not catching that, I am admittedly not super well-versed with flex box.  😬

File: src/components/header/header.module.css [hunk 3]
```diff
@@ -1,9 +1,10 @@
 .container {
   display: flex;
-  justify-content: space-between;
-  padding: 0 var(--size-4) var(--size-2);
-  margin: 0;
+  justify-content: center;
```
Comment (thescientist13) [inline src/components/header/header.module.css:3]: The padding / margin removal has made things a bit more cramped at the bottom of the header
Screenshot 2024-10-18 at 12 44 08 PM

Made [one small tweak](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/pull/113/commits/a0787c6bf7a4c9fa55e6f14225a2d103b089adb4) to retain some of that and now I think we're good to go!
Screenshot 2024-10-18 at 12 44 13 PM

**Review by thescientist13 — APPROVED** [review]
Thank you sir! 🫡

**Pull request merged**